roles/nova-data/templates/etc/severspec/kvm_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe file('/var/lib/libvirt/images') do
```

Comment:
I'm not sure we actually use this directory for anything. I mean, it doesn't hurt to look at it, but I don't expect it to ever be used.